webui/react/src/components/kit/Avatar.tsx

```diff
@@ -47,34 +52,51 @@ export const getColor = (name = '', darkLight: DarkLight): string => {
 };
 
 const Avatar: React.FC<Props> = ({
-  displayName,
+  text,
   hideTooltip,
   noColor,
-  size = Size.Medium,
+  size = Size.Small,
   square,
+  textColor = 'white',
+  statusText,
+  inactive,
 }) => {
   const { ui } = useUI();
 
   const style = {
-    backgroundColor: noColor ? 'var(--theme-stage-strong)' : getColor(displayName, ui.darkLight),
-    borderRadius: square ? '10%' : '100%',
-    color: noColor ? 'var(--theme-stage-on-strong)' : 'white',
+    backgroundColor: noColor ? 'var(--theme-stage-strong)' : getColor(text, ui.darkLight),
+    color: noColor ? 'var(--theme-stage-on-strong)' : textColor,
   };
   const classes = [css.base, css[size]];
+  if (square) classes.push(css.square);
+  if (inactive) classes.push(css.inactive);
 
   const avatar = (
     <div className={classes.join(' ')} id="avatar" style={style}>
-      {getInitials(displayName)}
+      {getInitials(text)}
     </div>
   );
 
   return hideTooltip ? (
     avatar
   ) : (
-    <Tooltip content={displayName} placement="right">
+    <Tooltip content={statusText ? [text, statusText].join(' ') : text} placement="right">
```

Comment:
It'a a bit counter-intuitive to me that we are concating `text` and `statusText` to construct tooltip content, can we change the props to be something like `TooltipContent`, then if that's not provided we default to `text`?